tests/pytest/test.py

```diff
@@ -1058,6 +1058,7 @@ def testCopyCommand(env):
     """Test COPY command and make sure behavior of json keys is similar to hash keys"""
 
     env.skipOnCluster() 
+    env.skipOnVersionSmaller('6.2')
```

Comment:
Should probably be [2.0.10](https://github.com/RedisJSON/RedisJSON/releases/tag/untagged-f2412154313e0faf5fdc)
```suggestion
    env.skipOnVersionSmaller('2.0.10')
```